lib/blanks.ex

```diff
@@ -32,8 +32,8 @@ defmodule Blanks do
     |> elem(1)
   end
 
-  defp count(:___, acc), do: {node, acc+1}
-  defp count({:___, _, _}, acc), do: {node, acc+1}
```

Comment:
Check out the commit notes https://github.com/elixirkoans/elixir-koans/commit/a2e3fe6d93b14e9105238f85554c8661e8ad9c53